packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -468,6 +475,14 @@ export interface ClusterOptions extends CommonClusterOptions {
    * @default true
    */
   readonly prune?: boolean;
+
+  /**
+   * If set to true, the cluster handler functions will be placed in the private subnets
+   * of the cluster vpc, subject to the `vpcSubnets` selection strategy.
+   *
+   * @default false
+   */
+  readonly placeClusterHandlerInVpc?: boolean;
```

Comment:
It would make sense to use a similar prefix:

```suggestion
  readonly clusterHandlerVpc?: boolean;
```